catalog-be/src/main/java/org/openecomp/sdc/be/components/impl/ServiceBusinessLogic.java

```diff
@@ -882,10 +882,8 @@ public void setArtifactBl(ArtifactsBusinessLogic artifactBl) {
         if (lock) {
             try {
                 lockComponent(service.getUniqueId(), service, "Delete Forwarding Path on Service");
-            } catch (ComponentException e) {
+            } finally {
                 janusGraphDao.rollback();
```

Comment:
With this change, `janusGraphDao.rollback();` will be called all the time (not only on ComponentException). See https://www.ibm.com/support/pages/best-practice-catching-and-re-throwing-java-exceptions for proper way to address this issue.